dtfs-central-api/src/helpers/index.ts

```diff
@@ -1,11 +1,10 @@
 export { isNumber } from './isNumber';
-export { feeRecordCsvRowToSqlEntity } from './fee-record.helper';
+export { feeRecordCsvRowToSqlEntity } from './fee-record-helpers';
```

Comment:
Can we remove `-helpers` please, then all helper files have the same naming convention 😎